### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by ImRodry**
**Describe the changes in this pull request:**
Beep boop I'm dependabot.
Also removed an unnecessary dep and fixed some versions

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Undo the deletion of eqeqeq fix

**Comment by ImRodry** [comment]
Rebased, not removing the dep as it is unnecessary. This is good to go

**Review by EncryptedDev — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -51,8 +50,11 @@
...
 		"semver": "^7.3.5",
-		"source-map-support": "^0.5.19",
-		"terser": "^5.9.0",
-		"typescript": "^4.5.3"
+		"source-map-support": "^0.5.21",
+		"terser": "^5.10.0",
+		"typescript": "^4.5.5"
+	},
+	"dependencies": {
+		"eslint-plugin-eqeqeq-fix": "^1.0.3"
```
Comment (Bas950) [inline package.json:58]: Dev dependency?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ImRodry) [reply package.json:58]: like I said we're removing that, rhys added that not me

**Comment by EncryptedDev** [comment]
Inactive

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**